### ISSUE TITLE
fix

### ISSUE THREAD

**Issue by mumaguma** [CptKirklnd/sims4-tweaks#1]
Hi,
i run on vista/python 3. First menu returned "invalid choice" on any numerical input.
I suggest changing  option = input("> ") to  option = int(input("> ")), as in:

def promptedInput():
        while True:
                menu()
                option = int(input("> "))
                if option == 1:
                        setRatios()
                elif option == 2:
                        setAgeLengths()
                elif option == 3:
                        displayCurrentSettings()
                elif option == 4:
                        if (export() == 0):
                            return 0
                else:
                        print("Invalid choice.")
        return 0

**Comment by CptKirklnd** [comment]
Yeah I run on Python 2.7 still, which might be the problem. I'll update w/ your fix a bit later, thanks! :)

Sent from my iPhone

On Mar 9, 2016, at 9:25 AM, mumaguma <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi,
i run on vista/python 3. First menu returned "invalid choice" on any numerical input.
I suggest changing option = input("> ") to option = int(input("> ")), as in:

def promptedInput():
while True:
menu()
option = int(input("> "))
if option == 1:
setRatios()
elif option == 2:
setAgeLengths()
elif option == 3:
displayCurrentSettings()
elif option == 4:
if (export() == 0):
return 0
else:
print("Invalid choice.")
return 0

-

**Comment by mumaguma** [comment]
also "raw_input" should become "input" for Python 3.x  :-)
